renderer/src/components/SchedulerCard.jsx

```
@@ -150,19 +178,23 @@
         <button
-          disabled={!enabled}
+          disabled={!enabled || executing}
           className={`text-xs px-2 py-1 rounded flex items-center ${
-            enabled
+            enabled && !executing
               ? 'bg-indigo-50 text-indigo-700 hover:bg-indigo-100'
               : 'bg-gray-50 text-gray-400 cursor-not-allowed'
           }`}
-          onClick={() => {/* TODO: Implement execute scheduler functionality */}}
+          onClick={handleExecute}
         >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)